.github/workflows/release.yml

```
@@ -16,78 +16,78 @@ concurrency: ${{ github.workflow }}-${{ github.ref }}
 
-        - name: Publish ${{ inputs.tag || 'canary' }}
-          if: steps.changesets.outputs.published != 'true'
-          run: |
-            git checkout main
-            bun version:canary
-            bun release:canary --tag ${{ inputs.tag || 'canary' }}
-          env:
-            NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-            GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-            NPM_CONFIG_PROVENANCE: true
+      - name: Publish ${{ inputs.tag || 'canary' }}
```

actionlint (1.7.4)

property "tag" is not defined in object type {}

(expression)